controls/pcidss_4.yml

```diff
@@ -2469,16 +2469,16 @@ controls:
       status: manual
 
     - id: 9.4.4
-      title: 'Management approves all media with cardholder data that is moved outside
-        the facility (including when media is distributed to individuals).Media cannot
-        leave a facility without the approval of accountable personnel.Individuals
-        approving media movements should have the appropriate level of management
-        authority to grant this approval. However, it is not specifically required that
-        such individuals have “manager” as part of their title.'
+      title: Management approves all media with cardholder data that is moved outside the facility
+        (including when media is distributed to individuals).
+      description: |-
+        Media cannot leave a facility without the approval of accountable personnel. Individuals
+        approving media movements should have the appropriate level of management authority to
+        grant this approval. However, it is not specifically required that such individuals have
+        “manager” as part of their title.
```

Comment:
No smart quotes

Reply:
Thanks. Fixed in 4f0ef2801ac522052ab76ad05f3ab0e1d6eccba1